### PR TITLE
[RFR] Use new ng-admin version and update Babel

### PR DESCRIPTION
**Pull request by jpetitcolas**
Do not  merge until [https://github.com/marmelab/ng-admin/pull/959](https://github.com/marmelab/ng-admin/pull/959) is released.

### CODE REVIEW

**Comment by jpetitcolas** [comment]
Mergeable.

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "url": "git://github.com/marmelab/ng-admin-demo.git"
   },
   "dependencies": {
-    "angular": "1.4.12",
```
Comment (Kmaschta) [inline package.json]: shame

Comment (Kmaschta) [inline package.json]: image

Comment (Kmaschta) [inline package.json]: shame

File: webpack.config.js [hunk 2]
```diff
@@ -1,7 +1,11 @@
 var ExtractTextPlugin = require('extract-text-webpack-plugin');
+var path = require('path');
 
 module.exports = {
-    entry: ["./js/main.js", "./css/main.scss"],
+    entry: [
+        './js/main.js',
+        // './css/main.scss',
```
Comment (fzaninotto) [inline webpack.config.js]: commented out code

Comment (jpetitcolas) [inline webpack.config.js]: Indeed. Webpack setup was weird (we used the built files directly: if code broke and no rebuild, then this break was invisible). I added the `HtmlWebpackPlugin` to fix it.

**Comment by fzaninotto** [comment]
please update the `make data`, which still generates data in `data.js`

File: makefile [hunk 3]
```diff
@@ -3,18 +3,14 @@
...
-	@cp ./node_modules/ng-admin/build/ng-admin.min.js build/
-	@cp ./node_modules/ng-admin/build/ng-admin.min.js.map build/
-
-run: copy-ng-admin
+run:
 	@echo "**************************************************"
 	@echo "* open http://localhost:8080/webpack-dev-server/ *"
 	@echo "**************************************************"
-	@./node_modules/.bin/webpack-dev-server --host=0.0.0.0 --progress --colors --devtool cheap-module-inline-source-map --hot --inline
+	@./node_modules/.bin/webpack-dev-server --host=0.0.0.0 --progress --colors --hot --inline
```
Comment (fzaninotto) [inline makefile]: please re-add the sourcemap

File: webpack.config.js [hunk 4]
```diff
@@ -1,25 +1,38 @@
...
         filename: "main.js",
-        publicPath: "/build"
+        publicPath: "/"
     },
     module: {
         loaders: [
-            { test: /\.css$/, loader: "style!css" },
-            { test: /\.jsx?$/, exclude: /(node_modules|bower_components)/, loader: 'babel' },
-            { test: /node_modules[\\\/]admin-config[\\\/].*\.jsx?$/, loader: 'babel' },
+            { test: /\.js$/, loader: 'babel' },
```
Comment (fzaninotto) [inline webpack.config.js]: We should copy this example webpack config in the ng-admin doc

**Pull request merged**